core/ledger/kvledger/txmgmt/validation/tx_ops_test.go

```diff
@@ -66,7 +66,7 @@ func TestTxOpsPreparationValueUpdate(t *testing.T) {
 		version.NewHeight(1, 2))
 
 	db.ApplyPrivacyAwareUpdates(updateBatch, version.NewHeight(1, 2)) //write the above initial state to db
-	precedingUpdates := NewPubAndHashUpdates()
+	precedingUpdates := newPubAndHashUpdates()
```

Comment:
```suggestion
      precedingUpdates := newPublicAndHashUpdates()
```
Using a single term consistently would help. Applicable for all other usages of this API.